tasks/buildah.yaml

```diff
@@ -113,12 +113,30 @@ spec:
     workingDir: $(workspaces.source.path)
     securityContext:
       runAsUser: 0
+  - image: registry.access.redhat.com/ubi8/python-39:1-51
+    name: convert-cyclonedx-to-purl
+    script: |
+      #!/bin/python3
+      import sys
+      import json
+
+      with open("./sbom.json") as f:
+        data = json.load(f)
+
+      purls = [component["purl"] for component in data["components"] if "purl" in component]
```

Comment:
it's returning just list of packages, purl format is json.

Reply:
You should use `json.dump` when you write the file.

Reply:
Also, be mindful of the format we use in Cachito nowadays. I know we won't be able to make it 100% equal, but we can at least maintain the structure like:
```
{
    "image_contents": {
        "dependencies": [
            {
                "purl": "pkg:alpine/alpine-baselayout@3.2.0-r18?arch=x86_64&upstream=alpine-baselayout&distro=alpine-3.15.4"
            },
            {
                "purl": "pkg:alpine/alpine-keys@2.4-r1?arch=x86_64&upstream=alpine-keys&distro=alpine-3.15.4"
            },
            //...
         ]
     }
}
```

Reply:
used the format from @brunoapimentel  suggestion, please check